kind/README.md

```diff
@@ -15,7 +15,6 @@ The following programs should be manually installed:
 - [Helm](https://helm.sh/docs/intro/install/) 3.10.0 or greater must be installed and configured on your machine.
 - [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) tool for running local Kubernetes clusters 0.14.0 or newer must be installed on your machine.
 - [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) 1.26 or newer must be installed on your machine.
-- [wget](https://www.gnu.org/software/wget/) 1.21 must be installed on your machine.
```

Comment:
can you also remove wget reference from the latest https://github.com/IBM/data-prep-kit/blob/dev/kfp/doc/simple_transform_pipeline.md version

Reply:
Added thanks